<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusNotifierTest</class>
 <widget class="QDialog" name="StatusNotifierTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>692</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stat&amp;usNotifier Testap</string>
  </property>
  <property name="windowIcon">
   <iconset theme="plasma">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="JobTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Stat&amp;us Notifier</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Log&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="logEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;b&gt;Status&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="enabledCheck">
         <property name="text">
          <string>Enabled</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="statusAuto">
         <property name="text">
          <string>Automatic</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QRadioButton" name="statusPassive">
         <property name="text">
          <string>Passive</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QRadioButton" name="statusActive">
         <property name="text">
          <string>Ac&amp;tive</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QRadioButton" name="statusNeedsAttention">
         <property name="text">
          <string>&amp;NeedsAttention</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;b&gt;Icon&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;b&gt;ToolTip&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="tooltipText"/>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="tooltipSubtext"/>
       </item>
       <item row="13" column="1">
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QFormLayout" name="iconsLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Icon</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>AttentionIcon</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>OverlayIcon</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="iconLayout">
           <item>
            <widget class="QLineEdit" name="iconName"/>
           </item>
           <item>
            <widget class="QCheckBox" name="iconPixmapCheckbox">
             <property name="text">
              <string>Use Pixmap</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="attentionIconLayout">
           <item>
            <widget class="QLineEdit" name="attentionIconName"/>
           </item>
           <item>
            <widget class="QCheckBox" name="attentionIconPixmapCheckbox">
             <property name="text">
              <string>Use Pixmap</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="overlayIconLayout">
           <item>
            <widget class="QLineEdit" name="overlayIconName"/>
           </item>
           <item>
            <widget class="QCheckBox" name="overlayIconPixmapCheckbox">
             <property name="text">
              <string>Use Pixmap</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;Jobs</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;b&gt;Job Control&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;b&gt;Progress&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QProgressBar" name="jobProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;b&gt;Naming&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="nameText"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="subText"/>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="speedSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="jobEnabledCheck">
         <property name="text">
          <string>Job Started</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
